File: src/main/java/pl/training/workshop/m4/s07_movemethod/step2/Booking.java
package pl.training.workshop.m4.s07_movemethod.step2;

/**
 * Rezerwacja jednego miejsca.
 *
 * @param seat numer miejsca; {@code Integer}, bo tak przychodzi ze starego API
 */
public record Booking(String id, Screening screening, Integer seat) {
}

File: typescript/src/workshop/m4/s07_movemethod/step2/Booking.ts
import type { Screening } from './Screening.js';

/**
 * Rezerwacja jednego miejsca.
 *
 * @param seat numer miejsca (numer z sali, nie pozycja na liście wolnych miejsc)
 */
export class Booking {
  constructor(readonly id: string, readonly screening: Screening, readonly seat: number) {}
}
